specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -6086,6 +6224,13 @@
             "$ref": "#/definitions/HealthError"
           }
         },
+        "failoverHealthErrorDetails": {
```

Comment:
The health summary detail is the summary of the container level resource. The purpose of this is to give a dashboard kind of view at the container level resource like vault/fabric. As stated earlier since we have discovered resources only adding a new property shouldn't be a breaking change.